org.lemonade_emu.lemonade.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.lemonade_emu.lemonade</id>
+  <name>Lemonade</name>
+  <summary>An Open Source Nintendo 3DS emulator</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+  <description>
+    <p>Lemonade is an open-source Nintendo 3DS emulator/debugger "fork" written in C++. It is written with portability in mind, with builds actively maintained for Windows, Linux and macOS.</p>
+    <p>Lemonade plans to add new features/enhacments on top of the __base__ emulator which was taken down by their team</p>
+  </description>
+  <launchable type="desktop-id">org.lemonade_emu.lemonade.desktop</launchable>
+  <provides>
+    <binary>lemonade-qt</binary>
+    <id>lemonade-qt.desktop</id>
+  </provides>
+  <url type="homepage">https://lemonade-emu.github.io/</url>
+  <url type="contact">https://discord.gg/kKKantKn</url>
+  <screenshots>
+    <screenshot type="default"><image>https://github.com/Lemonade-emu/Lemonade/blob/master/assets/lemonade_flathub-2.png</image></screenshot>
+    <screenshot><image>https://github.com/Lemonade-emu/Lemonade/blob/master/assets/lemonade_flathub-1.png</image></screenshot>
+    <screenshot><image>https://github.com/Lemonade-emu/Lemonade/blob/master/assets/lemonade_flathub-3.png</image></screenshot>
```

Comment:
1. this link don't link to the image.
2. they should NEVER point at a branch but at a specific commit

Reply:
Im fixing that , will push shortly